src/components/NavBar.jsx

```diff
@@ -1,81 +1,67 @@
 import MenuToggle from './MenuToggle';
 import Loader from '../components/Loader';
 import SideBar from './Sidebar';
 
-const sidebar = {
-  open: (height = 1000) => ({
-    clipPath: `circle(${height * 2 + 200}px at 40px 40px)`,
-    transition: {
-      type: 'spring',
-      stiffness: 20,
-      restDelta: 2,
-    },
-  }),
-  closed: {
-    clipPath: 'circle(30px at 40px 40px)',
-    transition: {
-      delay: 0.5,
-      type: 'spring',
-      stiffness: 400,
-      damping: 40,
-    },
-  },
-};
-
 export default function NavBar() {
-  const [isOpen, toggleOpen] = useCycle(false, true);
+  const [isInputOpen, setInputOpen] = useState(false);
+  const [isOpen, setIsOpen] = useState(false);
   const [navVisible, setNavVisible] = useState(false);
+
+  const handleMouseEnter = () => {
+    setInputOpen(true);
+  };
+
+  const handleMouseLeave = () => {
+    setInputOpen(false);
+  };
+
   useEffect(() => {
     const timeoutId = setTimeout(() => {
       setNavVisible(true);
     }, 1000 * 5);
     return () => clearTimeout(timeoutId);
   }, []);
 
-  const effect = isOpen
-    ? 'absolute w-80 h-screen top-1 left-0 bg-gradient-to-r from-slate-500-opacity-30 via-slate-600-opacity-80 -z-50'
-    : 'absolute w-80 top-11 left-0 -z-50';
-
   return (
     <>
       <Loader />
       {navVisible && (
-        <nav className="via-slate-800-opacity-80 to-slate-500-opacity-30 absolute z-10 flex w-full items-center justify-between bg-gradient-to-b from-dark px-7 py-5 opacity-95">
+        <nav className="via-slate-800-opacity-80 to-slate-500-opacity-30 absolute z-10 flex w-full items-center justify-between bg-gradient-to-b from-dark px-7 py-12 opacity-95">
           {/*  menu burger */}
-          <motion.div
-            initial={false}
-            animate={isOpen ? 'open' : 'closed'}
-            custom="100%"
-            className="w-1/3"
-          >
-            <motion.div className={effect} variants={sidebar} />
-            <SideBar />
-            <MenuToggle toggle={() => toggleOpen()} />
-          </motion.div>
+          <MenuToggle isOpen={isOpen} setIsOpen={setIsOpen} />
           <div className="group relative flex cursor-pointer justify-center">
             {/*  logo */}
             <Link to={'/'}>
               <Logo />
             </Link>
           </div>
           {/*  input */}
-          <div className="group relative flex w-1/3 justify-end">
+          <form
+            type="texte"
+            className="relative flex w-1/3 justify-end"
+            onMouseLeave={handleMouseLeave}
+          >
             <input
               type="text"
-              className="h-10 w-10 rounded-full p-4 opacity-0 shadow outline-none transition-all duration-500 ease-in-out group-hover:w-28 group-hover:opacity-100 group-hover:sm:w-52"
+              className={`h-10 w-10 rounded-full p-4 shadow outline-none transition-all duration-500 ease-in-out
+    ${isInputOpen ? 'w-28 sm:w-52' : 'opacity-0'}`}
```

Comment:
Saut de ligne au milieu du classname
```suggestion
              className={`h-10 w-10 rounded-full p-4 shadow outline-none transition-all duration-500 ease-in-out ${isInputOpen ? 'w-28 sm:w-52' : 'opacity-0'}`}
```

Reply:
erff avec le npm run format il saute des lignes ...

Reply:
Non c'est bon maintenant 👍